.eslintrc

```diff
@@ -37,6 +37,8 @@
   "rules": {
     "prettier/prettier": ["error"],
     "react/prop-types": 0,
+    "react/no-multi-comp": 0,
+    "react/jsx-wrap-multilines": "off",
```

Comment:
Wouldn't this rule lead to messy markup?

Reply:
Not really, as far as I know it's only forcing the developer to use parentheses for better readability. I think it's a bit of an overkill for this project as we develop small components, but that's my opinion. I'm okay with re-enable this again :)